stdlib/public/stubs/Stubs.cpp

```diff
@@ -532,10 +532,19 @@ __swift_bool _swift_stdlib_getCurrentStackBounds(__swift_uintptr_t *outBegin,
   *outEnd = highLimit;
   return true;
 
-#elif defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__ANDROID__) || defined(__linux__)
+#elif defined(__OpenBSD__)
+  stack_t sinfo;
+  if (pthread_stackseg_np(pthread_self(), &sinfo) != 0) {
```

Comment:
I haven't profiled closely or anything, but vaguely reviewing the implementation: https://github.com/openbsd/src/blob/master/lib/librthread/rthread_np.c#L70 

Apparently it will either inspect the thread data it already has, or it's the initial stack and it's grabbing the process arguments via `sysctl` to base the stack pointer on and `getrlimit` for size, which suggests that there might be a slight initial cost on first call and subsequent calls are cheap.

Reply:
Sounds fine to me for the test suite's use case.